### PR TITLE
test: Begin porting influxrpc tests to ng end to end tests

### PR DESCRIPTION
**Pull request by alamb**
# Rationale:
Towards https://github.com/influxdata/influxdb_iox/issues/3958 where we can remove OG

# Changes:
1. Ports the first few gRPC tests over to new end to end framework, demonstrate intended pattern

### CODE REVIEW

File: influxdb_iox/tests/end_to_end_ng_cases/querier/influxrpc.rs [hunk 1]
```diff
@@ -0,0 +1,448 @@
+//! Tests for influxrpc / Storage gRPC endpoints
+
+mod data;
+mod dump;
+mod exprs;
+
+use std::sync::Arc;
```
Comment (alamb) [inline influxdb_iox/tests/end_to_end_ng_cases/querier/influxrpc.rs:7]: This is the port of https://github.com/influxdata/influxdb_iox/blob/7a43738/influxdb_iox/tests/end_to_end_cases/influxrpc.rs#L24-L317

File: test_helpers_end_to_end_ng/src/mini_cluster.rs [hunk 2]
```diff
@@ -22,25 +22,65 @@ pub struct MiniCluster {
...
-            org,
-            bucket,
+            org_id,
+            bucket_id,
             namespace,
             ..Self::default()
         }
     }
 
+    /// Create a "standard" MiniCluster that has a router, ingester,
```
Comment (alamb) [inline test_helpers_end_to_end_ng/src/mini_cluster.rs:44]: as part of https://github.com/influxdata/influxdb_iox/issues/4368 plan to refactor all tests that use a common mini cluster configuration to use the same mini cluster for performance reasons -- this is the first part of that.

File: influxdb_iox/tests/end_to_end_ng_cases/querier/influxrpc.rs [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+    let mut cluster = MiniCluster::create_standard(database_url).await;
+
+    StepTest::new(&mut cluster, vec![Step::Custom(custom)])
+        .run()
+        .await
+}
+
+#[tokio::test]
+async fn read_filter() {
+    let generator = Arc::new(data::DataGenerator::new());
```
Comment (alamb) [inline influxdb_iox/tests/end_to_end_ng_cases/querier/influxrpc.rs:84]: there is a non trivial amount of `async` ceremony in these tests, but I think they are still pretty readable

**Review by NGA-TRAN — APPROVED** [review]
I do not look into the details but I saw corresponding tests moved from end_to_end_cases to end_to_end_ng_cases

**Comment by alamb** [comment]
Thanks @NGA-TRAN  -- I am merging this PR so I can continue with some other changes I have in the works

**Pull request merged**